src/scripts/location-select.js

```diff
@@ -131,6 +210,26 @@ fetch('/cities.json').then((res) => res.json()).then((cities) => {
             });
     }
 
-    $input?.addEventListener('change', onChange);
-    onChange();
-});
\ No newline at end of file
+    /**
+     * Triggers several functions when a location is selected.
+     * 
+     * @param {Element} $btn - The element selected by the user
+     */
+    function userChosen($btn = $locations.querySelector('button[tabIndex="0"]')) {
```

Comment:
nit: `locationSelected`?

Reply:
I'm being specific about when the user chooses the location versus when the system does. I could align the function names to reflect that.